datafusion/core/tests/sql/subqueries.rs

```diff
@@ -352,6 +351,21 @@ async fn non_aggregated_correlated_scalar_subquery_with_single_row() -> Result<(
         "\n\nexpected:\n\n{expected:#?}\nactual:\n\n{actual:#?}\n\n"
     );
 
+    // TODO infer nullability in the schema has bug
```

Comment:
is this tracked by a ticket? It probably would be good to put the link URL in the code do so we don't lose track of it

Reply:
Sure, will add one.